gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -256,6 +272,102 @@ void GazeboRosWheelSlip::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
       }
     }
   }
+
+  double zero_wheel_spin_tolerance = 1.0e-3;
+  if (!_sdf->HasElement("zero_wheel_spin_tolerance")) {
```

Comment:
nit: rename `zero_wheel_spin_tolerance` -> `wheel_spin_tolerance`

Reply:
Fixed in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1331/commits/53c792ba286876e9e477153ee02cdb5f8fa825e6.